.xperior/testds/motr-single_tests.yaml

```diff
@@ -490,7 +490,13 @@ Tests:
   - id       : 37protocol
     script   : 'm0 run-st 37protocol'
     dir      : src/scripts
-    executor : Xperior::Executor::MotrTest
+    # this ST checks checksum of the BE structures during
+    # developmental phase they are bound to fail so code
+    # changes are required but ST has to be in disabled
+    # state for now when the release is in GA phase we will
+    # enable the ST again to ensure there will be no changes
+    # in BE structures.
```

Comment:
```suggestion
    # this ST confirms that no BE structures are changed
    # in the new version of the code since that would 
    # create a mismatch of BE structures and thus cause 
    # corruption in case of upgrades. During development 
    # phase the BE structures are assumed to be modified
    # and to avoid this ST from failing it is better to disable
    # the ST until a GA release is done
```